src/Microsoft.AspNet.Mvc.Core/MvcOptions.cs

```diff
@@ -64,6 +68,16 @@ public AntiForgeryOptions AntiForgeryOptions
         public List<OutputFormatterDescriptor> OutputFormatters { get; private set; }
 
         /// <summary>
+        /// Sets the mapping for output format specified in URL (extension) and content type.
+        /// </summary>
+        /// <param name="format">URL extension for output format.</param>
+        /// <param name="contentType">Content type mapping to the format.</param>
+        public void AddFormatMapping(string format, MediaTypeHeaderValue contentType)
```

Comment:
Remove the method. Change OutputFormatterOptions --> FormatterMappings, FormatterMediaTypeMappings?